.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,20 +16,22 @@ This section describes how to determine whether a blueprint has enough value to
 This section describes how to determine whether the blueprint has the right technical quality to be included in the central XebiaLabs blueprints repository.
 
 - [ ] The branch from which the PR is created is up-to-date with the `development` branch.
+- [ ] There is a `README.md` file at the root of the blueprint folder that describes the blueprint, using the standard README template.
```

Comment:
This is already in the DOD. See the next line. :-)

Reply:
I took a shot at merging the two. Haven't had a chance to chat with Deepu about it yet.